types/xk6-sql-browser/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "npm-naming": false
```

Comment:
```suggestion
```
This needs to be removed and if any error fixed.

Reply:
Done.